src/analysis_and_optimization/Debug_data_generation.ml

```diff
@@ -300,7 +300,7 @@ let generate_json_entries (name, expr) : string * t =
           [%message "Could not evaluate expression " (e : Expr.Typed.t)] in
   (name, expr_to_json expr)
 
-let print_data_prog data =
+let print_declarations_json data =
```

Comment:
This function does not print declarations; it takes declarations as input and produces a JSON string of conforming values. "Creating a string" counts as "printing" (*I guess...*) and while the values aren't necessarily "data" in the sense Stan programs use that word, they're still data in a more general sense. I'd say the original name, although not prefect, was more fitting.

Alternative idea:
```ocaml
let gen_values_json decls =
```